src/modules/events/components/eventCard/index.js

```diff
@@ -1,7 +1,63 @@
-import React, { Component } from "react";
+import React, { Component }  from 'react'
+import {Link} from 'react-router-dom'
+import styles from "./style.module.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import
+ {   faMapMarkerAlt,
+     faCalendarCheck ,
+     faClock
+  } from "@fortawesome/free-solid-svg-icons";
 
-export default class EventCard extends Component {
-  render() {
-    return <div></div>;
-  }
+class EventCard extends Component{ 
+
+  render(){
+  
+    const {events} = this.props ;
+
+    return (
+           <div className={styles["event-card_content"]}>
+              <figure className={styles["events-card_cover"]} key={events}>
+                  <Link to={'/' + events.id}>
+                      <img  
+                        alt={`event ${events.title}`}
+                        src={events.cover}
+                        key={events.id} 
+                        />
+                  </Link>
+                  <p className={styles['activ-upcoming_event']}>
+                    {events.status === "upcoming"?<div>
+                     <FontAwesomeIcon icon={faClock}
+                        className={styles["activ-upcoming-event_icon"]}/></div>: ''}
+                  </p>
+                  <figcaption >
+                      <h3>
+                        <Link to={'/' + events.id} className={styles["event-card_title"]}>
+                          {events.title}
+                          </Link>
+                      </h3>
+                      <p className={styles["event-card_description"]}>
+                          {events.description}
```

Comment:
The description should be limited to a number of words, e.g: 20 words.

Reply:
I'm used lines ellipsis to limited it

Reply:
Great